rehfeldmedical/openshift-centos-playbooks
# SCAUT-365: Make prepare for openshift into a role.

This PR converts the prep-server playbook into a re-usable set of roles. The os-update role will install and OS updates, while the prepare-openshift role will perform pre-requisite changes for Openshift installation.

Changes
====
1. Converted prep-server playbook into os-update and prepare-openshift roles.
2. Made prep-server playbook use the new roles.
3. Made file limit part of the role optional, and reversable.

Testing
====
1. Create a new VM via `vagrant up` in the vagrant playground with a `ansible.cfg` and `hosts.local` created.
2. Login to VM, confirm IP address and add to `hosts.local`
3. Confirm there are no updates pending by `yum update`
4. Confirm that file limits are increased by `ulimit -n`
5. Confirm that nofile limit is increase by `sysctl fs.file-max`. This limit may be below 64000 due to kernel limits
